### PR TITLE
Port source for us-wa

### PR DESCRIPTION
**Pull request by lazd**
## Summary

Port US-WA scraper

## Todo

* [x] Test current scraper
* [x] Test old scrapers against migrated cached files

### CODE REVIEW

**Comment by jzohrab** [comment]
I'll try running the current integration tests with this branch, to see what happens.  I'll need to work on a test to see if the scrapers work for past cached files.

**Comment by jzohrab** [comment]
Current integration test passes for this:

```
$ git checkout upstream/us-wa -b us-wa
Branch 'us-wa' set up to track remote branch 'us-wa' from 'upstream'.
Switched to a new branch 'us-wa'
$ npm run test:integration
... lots of activity ...
⚠️  us-wa: stale data for Yakima County
   Scrape date: 2020-04-22
   Last updated: 4/20/2020, 6:00:00 PM
    Scrape-us-wa-2020-04-22: 382.522ms
    ✔ us-wa ok

  Teardown

    ✔ Removed temp directory
    ✔ Sandbox closed

  Testing source: us-wa

    ✔ null error "null"
    ✔ completed successfully
    ✔ crawled
    ✔ scraped
    ✔ got data


  total:     10
  passing:   10
  duration:  6s
```

**Comment by lazd** [comment]
I tested the source with the relevant date ranges for the old scrapers:

```
./start -c us-wa -d 2020-03-13
./start -s us-wa -d 2020-03-13
./start -s us-wa -d 2020-03-19
./start -s us-wa -d 2020-03-23
./start -s us-wa -d 2020-03-31
```

All of them seemed to work!

File: src/shared/sources/us/wa/index.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+      crawl: [
+        {
+          type: 'json',
+          url: 'https://services8.arcgis.com/rGGrs6HCnw87OFOT/arcgis/rest/services/CountyCases/FeatureServer/0/query?f=json&where=(CV_State_Cases%3E0)&returnGeometry=false&outFields=*&orderByFields=CNTY_NAME%20asc'
+        }
+      ],
+      scrape (data, scrapeDate) {
+        const counties = []
+        data.features.forEach(item => {
+          const cases = item.attributes.CV_PositiveCases
```
Comment (jzohrab) [inline src/shared/sources/us/wa/index.js:187]: We should have a library for json schema validation.  That's not a blocker for this PR, though.

**Pull request merged**